From d75128e4dbe194909db24628b2c4774bb9ca0a21 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@kpanic.de>
Date: Thu, 14 Mar 2019 10:00:43 +0100
Subject: [PATCH 8/9] healthd: make periodic battery status a debug message

Battery status is printed to the kernel log as a warning every
10 seconds.
[  276.605006] healthd: battery l=88 v=4235 t=29.6 h=2 st=2 c=0 chg=u
[  286.662791] healthd: battery l=88 v=4238 t=29.5 h=2 st=2 c=0 chg=u
[  296.707097] healthd: battery l=88 v=4238 t=29.5 h=2 st=2 c=0 chg=u

That's a debug statement and not a warning.

Change-Id: I45cfdeb150de59e2ef68749e0ab400da3599d156
---
 healthd/BatteryMonitor.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/healthd/BatteryMonitor.cpp b/healthd/BatteryMonitor.cpp
index cde2f9711..a76ac03c0 100644
--- a/healthd/BatteryMonitor.cpp
+++ b/healthd/BatteryMonitor.cpp
@@ -362,7 +362,7 @@ bool BatteryMonitor::update(void) {
                  props.chargerUsbOnline ? "u" : "",
                  props.chargerWirelessOnline ? "w" : "");
 
-        KLOG_WARNING(LOG_TAG, "%s\n", dmesgline);
+        KLOG_DEBUG(LOG_TAG, "%s\n", dmesgline);
     }
 
     healthd_mode_ops->battery_update(&props);
-- 
2.17.1

